src/core/Akka.API.Tests/verify/CoreAPISpec.ApproveCore.DotNet.verified.txt

```diff
@@ -1725,6 +1725,7 @@ namespace Akka.Actor
         public int LogDeadLetters { get; }
         public bool LogDeadLettersDuringShutdown { get; }
         public System.TimeSpan LogDeadLettersSuspendDuration { get; }
+        public Akka.Event.LogFilterEvaluator LogFilter { get; }
```

Comment:
New setting - this is where the log filter actually lives. Any custom logging plugin can access it this way.